sdcm/utils/replication_strategy_utils.py

```diff
@@ -25,7 +25,7 @@ def get(cls, node: BaseNode, keyspace: str):
         return ReplicationStrategy.from_string(create_ks_statement)
 
     def apply(self, node: BaseNode, keyspace: str):
-        cql = f"ALTER KEYSPACE {keyspace} WITH replication = {self}"
+        cql = f'ALTER KEYSPACE "{keyspace}" WITH replication = {self}'
```

Comment:
I suggest running it via cql_qoute function, which is intended exactly for those cases

Reply:
Then you won't need to change all the unit tests